dashboard/src/instrumentation-node.ts

```diff
@@ -0,0 +1,78 @@
+import { context } from "@opentelemetry/api";
+import { AsyncLocalStorageContextManager } from "@opentelemetry/context-async-hooks";
+import { getRPCMetadata, RPCType, setRPCMetadata } from "@opentelemetry/core";
+import { PrometheusExporter } from "@opentelemetry/exporter-prometheus";
+import { HostMetrics } from "@opentelemetry/host-metrics";
+import { registerInstrumentations } from "@opentelemetry/instrumentation";
+import { HttpInstrumentation } from "@opentelemetry/instrumentation-http";
+import { RuntimeNodeInstrumentation } from "@opentelemetry/instrumentation-runtime-node";
+import {
+  detectResources,
+  envDetector,
+  hostDetector,
+  processDetector,
+  resourceFromAttributes,
+} from "@opentelemetry/resources";
+import { MeterProvider } from "@opentelemetry/sdk-metrics";
+import {
+  ATTR_SERVICE_NAME,
+  ATTR_SERVICE_VERSION,
+} from "@opentelemetry/semantic-conventions";
+import { IncomingMessage } from "http";
+
+// manually setting a context manager to replace the no-op context manager
+context.setGlobalContextManager(new AsyncLocalStorageContextManager());
+
+const exporter = new PrometheusExporter({
+  port: 9464,
```

Comment:
Can this port passed as an environment variable? You can set a default value but some times we may need to change the port.